config.toml.sample

```diff
@@ -226,6 +226,20 @@ type = "autocompound"
 featured = true
 status = true
 
+
+[protocols.osmosis-lp]
```

Comment:
I assume that we need to add frontend data for this source

Reply:
it would be nice if you can help with that. or I'll do it just later

Reply:
```suggestion
[protocols.osmosis-lp.frontend_data]
link = "https://app.osmosis.zone/pools"
link_text = "Go to Osmosis LP"
chain_name = "osmosis"
[protocols.osmosis-lp.assets.dATOM.frontend_data]
visible = true
strategy = "dATOM/ATOM"
description = "LP on Osmosis"
type = "correlated"
featured = false
status = true
```
I am not sure maybe smth like this

Reply:
```suggestion
[protocols.osmosis-lp]
...

[protocols.osmosis-lp.frontend_data]
link = "https://app.osmosis.zone/pools"
link_text = "Go to Osmosis LP"
chain_name = "osmosis"
[protocols.osmosis-lp.assets.dATOM.frontend_data]
visible = true
strategy = "dATOM/ATOM"
description = "LP on Osmosis"
type = "correlated"
featured = false
status = true
```
I am not sure maybe smth like this

Reply:
thanks a lot!